From 9fed24c031a885264a9249eed3b6c654c32ce139 Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Wed, 27 Feb 2013 01:05:34 -0500
Subject: [PATCH 2/2] udhcpc: tweak math shell style with the metric var

Some shells (like dash) are lame and omit the POSIX increment/decrement
feature (because it is listed as optional).  Tweak the shell script to
work in all POSIX variants.

Signed-off-by: Mike Frysinger <vapier@gentoo.org>
---
 examples/udhcp/simple.script | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/examples/udhcp/simple.script b/examples/udhcp/simple.script
index d42f2d3..0397e50 100755
--- a/examples/udhcp/simple.script
+++ b/examples/udhcp/simple.script
@@ -29,7 +29,8 @@ case "$1" in
 			metric=0
 			for i in $router ; do
 				echo "Adding router $i"
-				route add default gw $i dev $interface metric $((metric++))
+				route add default gw $i dev $interface metric $metric
+				: $(( metric += 1 ))
 			done
 		fi
 
-- 
1.7.12.4

